### ISSUE TITLE
Problem when training Multiband MelGAN

### ISSUE THREAD

**Issue by tienanh-1999** [TensorSpeech/TensorFlowTTS#247]
Some early steps, it is okay
![1](https://user-images.githubusercontent.com/47650134/92219259-858f8e80-eec4-11ea-8d28-dd6f097ab407.png)
![9](https://user-images.githubusercontent.com/47650134/92219401-b8398700-eec4-11ea-8c63-c3be313e56f8.png)

After that, I got a problem
![1](https://user-images.githubusercontent.com/47650134/92219466-cdaeb100-eec4-11ea-9d7f-f940f04687ac.png)
![1](https://user-images.githubusercontent.com/47650134/92219508-d7381900-eec4-11ea-9f54-8666ea2e94aa.png)

I also fix that like https://github.com/TensorSpeech/TensorFlowTTS/issues/113, but it doesn't work 
![image](https://user-images.githubusercontent.com/47650134/92219910-68a78b00-eec5-11ea-9904-fa35b9cf5742.png)

Thank you !

**Comment by dathudeptrai** [comment]
@tienanh-1999 try https://github.com/TensorSpeech/TensorFlowTTS/tree/master/examples/multiband_pwgan instead.

**Comment by dathudeptrai** [comment]
```
sub_sc_loss = tf.where(sub_sc_loss >= 10.0, 0.0, sub_sc_loss)
sub_mag_loss = tf.where(sub_mag_loss >= 10.0, 0.0, sub_mag_loss)
full_sc_loss = tf.where(full_sc_loss >= 10.0, 0.0, full_sc_loss)
full_mag_loss = tf.where(full_mag_loss >= 10.0, 0.0, full_mag_loss)
gen_loss = 0.5 * (sub_sc_loss + sub_mag_loss) + 0.5 * (
     full_sc_loss + full_mag_loss
)
```

try this. Let read the loss function here (https://github.com/TensorSpeech/TensorFlowTTS/blob/master/examples/multiband_melgan/train_multiband_melgan.py#L109)

**Comment by tienanh-1999** [comment]
thank you, I will try this !

**Comment by tienanh-1999** [comment]
I solved this by decreasing the learning rate (similar to the paper). 
Thank you !

**Comment by dathudeptrai** [comment]
> I solved this by decreasing the learning rate (similar to the paper).
> Thank you !

okay, let share some results if you can :)).

**Comment by tienanh-1999** [comment]
> > I solved this by decreasing the learning rate (similar to the paper).
> > Thank you !
> 
> okay, let share some results if you can :)).

https://drive.google.com/drive/folders/1IST9UElMdxFj2SOJwQbUcNDp1PJ4Gdju?usp=sharing, here is the result after 400k steps :D

**Comment by dathudeptrai** [comment]
> > > I solved this by decreasing the learning rate (similar to the paper).
> > > Thank you !
> > 
> > 
> > okay, let share some results if you can :)).
> 
> https://drive.google.com/drive/folders/1IST9UElMdxFj2SOJwQbUcNDp1PJ4Gdju?usp=sharing, here is the result after 400k steps :D

it's good :)), let see the quality at 1M steps.
